docs/core/tutorials/using-on-mac-vs.md

```diff
@@ -17,7 +17,7 @@ Visual Studio for Mac provides a full-featured Integrated Development Environmen
 
 ## Prerequisites
 
-See the [.NET Core dependencies and requirements](../install/dependencies.md?tabs=netcore30&pivots=os-macos) topic.
+See the [.NET Core dependencies and requirements](../install/dependencies.md?tabs=netcore30&pivots=os-macos) article.
```

Comment:
Is this article specific to core 3.0 ?
If not, I'd make the tab .NET Core 3.1, or maybe even remove the `tabs=` so it points to the latest version.